Dockerfile

```diff
@@ -5,26 +5,39 @@ MAINTAINER docker@stefan-van-essen.nl
 ENV LANG='en_US.UTF-8' LANGUAGE='en_US.UTF-8' TERM='xterm' DOCKER_HOST='docker'
 
 RUN apk -U --no-cache add \
-    curl \
-    git \
-    docker \
-    make \
-    nodejs-current-npm \
-    php7 \
-    php7-ctype \
-    php7-dom \
-    php7-json \
-    php7-mbstring \
-    php7-mcrypt \
-    php7-pdo_mysql \
-    php7-pdo_pgsql \
-    php7-pdo_sqlite \
-    php7-phar \
-    php7-tokenizer \
-    php7-session \
-    php7-simplexml \
-    php7-zip \
-    php7-zlib \
+        curl \
+        git \
+        docker \
+        make \
+        automake \
+        nodejs-current-npm \
+        php7 \
+        php7-ctype \
+        php7-dom \
+        php7-json \
+        php7-mbstring \
+        php7-mcrypt \
+        php7-pdo_mysql \
+        php7-pdo_pgsql \
+        php7-pdo_sqlite \
+        php7-phar \
+        php7-tokenizer \
+        php7-session \
+        php7-simplexml \
+        php7-zip \
+        php7-zlib \
+        alpine-sdk \
+        nasm \
+        autoconf \
+        build-base \
+        zlib \
+        zlib-dev \
+        libpng \
+        libpng-dev \
+        libwebp \
+        libwebp-dev \
+        libjpeg-turbo \
+        libjpeg-turbo-dev \
```

Comment:
Please fix your crappy indenting of 4 spaces to the Glorious Master Indenting that is 2 spaces.

All hail 2 space indenting, ruler of the codebase!